### ISSUE TITLE
KeyError: 'data/miniimagenet/images/n0185567200000776.jpg'

### ISSUE THREAD

**Issue by InugYoon** [icoz69/CEC-CVPR2021#4]
Hello,

When I tried to run train.py script, 
modification of PROJECT='cec' & dataset default as 'miniimagenet' occurs such error.

It occurs on miniimagenet.py line 80, proceeding data2label[img_path]

in miniimagenet.py, @ class MiniImageNet
self.data2label is created (44line) according to csv files (27, 36~44 lines)

It seems the error occured due to mismatch between contents of session files and csv files

Have you proceed any modifications on it?

**Comment by guanghuili98** [comment]
Hello, I also encountered the same problem, do you have any new progress?

**Comment by Nan-S** [comment]
Hi,
We didn't do any modification on the session files and csv files. I cannot reproduce the error. I check on the session file and csv file, there is no mismatch.
line 3306 train.csv:      n0185567200000776.jpg,n01855672
line 3001 session_1.txt: MINI-ImageNet/train/n01855672/n0185567200000776.jpg

Can you double check on your side?

**Comment by InugYoon** [comment]
Hello, I solved the issue. 
When I downloaded the miniimagenet from 'here', as written in the Readme of this repo, 
the csv files are split to train/val/test. 
However, when I see  the dataset/index_list in the github here, the csv files are split to train/test
The mismatch comes from here.
So as I replaced the csv files on the data/miniimagenet/~ from the downloaded version into the repo version, 
the issue was removed. 

Thanks!
